src/Common/LRUCache.h

```diff
@@ -326,13 +377,33 @@ class LRUCache
             }
 
             const auto & cell = it->second;
+            auto evict_status = evict_policy.canRelease(*cell.value);// in default, it is CAN_EVITCT
```

Comment:
Not sure about this. Calling `canRelease` seems to mean that the result is bool `can_release`: can or can't. But now it returns states.